File: client/Client/Network/Messages/Game/MoveActorMessage.cs
using LoadRunnerClient.MapAndModel;

namespace LoadRunnerClient.Messages
{
    /// Author Florian Ortmann
	public class MoveActorMessage
    {
        public const string TYPE = "MoveActorMessage";

        public MoveActorMessage()
        {
        }

        /// <summary>
        /// MoveActorMessage Constructor
        /// </summary>
        /// <param name="uuid"></param>
        /// <param name="xy"></param>
        /// <param name="state"></param>
        public MoveActorMessage(string uuid, Vector2 xy, GameCharacterState state)
        {
            this._uuid = uuid;
            this._newPosition = xy;
            this._state = state;
        }

        private string _uuid;

        public string uuid
        {
            get => _uuid;
            set => _uuid = value;
        }

        private Vector2 _newPosition;

        public Vector2 newPosition
        {
            get => _newPosition;
            set => _newPosition = value;
        }

        private GameCharacterState _state;

        public GameCharacterState state
        {
            get => _state;
            set => _state = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/MoveActorMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;


import de.hsrm.mi.swtpro03.server.game.model.GameCharacterState;
import de.hsrm.mi.swtpro03.server.game.model.Vector2;

public class MoveActorMessage {

	public static final String TYPE = "MoveActorMessage";

	private String uuid;
	private Vector2 newPosition;
	private GameCharacterState state;

	public MoveActorMessage() {
	}

	public MoveActorMessage(String uuid, Vector2 newPosition, GameCharacterState state) {
		this.uuid = uuid;
		this.newPosition = newPosition;
		this.state = state;
	}

	public String getUuid() {
		return uuid;
	}

	public void setUuid(String uuid) {
		this.uuid = uuid;
	}

	public Vector2 getNewPosition() {
		return newPosition;
	}


	public void setNewPosition(Vector2 newPosition) {
		this.newPosition = newPosition;
	}

	public GameCharacterState getState() {
		return state;
	}

	public void setState(GameCharacterState state) {
		this.state = state;
	}
}
